package.json

```diff
@@ -1,24 +1,43 @@
 {
-  "name": "jobsync",
+  "name": "next-js-launchpad",
```

Comment:
shouldn't the project name be 'jobsync' ?

Reply:
Yes, changed. Resolving.